runtime/queries/prql/highlights.scm

```diff
@@ -0,0 +1,140 @@
+[
+  (keyword_from)
+  (keyword_filter)
+  (keyword_derive)
+  (keyword_group)
+  (keyword_aggregate)
+  (keyword_sort)
+  (keyword_take)
+  (keyword_window)
+  (keyword_join)
+  (keyword_select)
+  (keyword_switch)
+  (keyword_inner)
+  (keyword_right)
+  (keyword_full)
+  (keyword_in)
+  (keyword_rolling)
+  (keyword_rows)
+  (keyword_expanding)
+  (keyword_let)
+  (keyword_prql)
+  (keyword_switch)
+  (keyword_append)
+  (keyword_remove)
+  (keyword_intersect)
+] @keyword
+
+(literal) @string
+
+(assignment
+  alias: (field) @field)
+
+alias: (identifier) @field
```

Comment:
```suggestion
  alias: (field) @variable.other.member)

alias: (identifier) @variable.other.member
```

Reply:
I guess in line 31, the last bracket should still close after `other.member` right?

Reply:
Yep right, I deleted too much in this suggestion 😅